rustls/src/manual/howto.rs

```diff
@@ -6,7 +6,7 @@ However, if your private key resides in a HSM, or in another process, or perhaps
 another machine, rustls has some extension points to support this:
 
 The main trait you must implement is [`sign::SigningKey`][signing_key]. The primary method here
-is [`choose_scheme`][choose_scheme] where you are given a set of [`SignatureScheme`s][sig_scheme] the client says
+is [`choose_scheme()`][choose_scheme] where you are given a set of [`SignatureScheme`s][sig_scheme] the client says
```

Comment:
reverting this for now:

```suggestion
is [`choose_scheme`][choose_scheme] where you are given a set of [`SignatureScheme`s][sig_scheme] the client says
```

It looks to me like links like this do not naturally show `()` for member functions:

    [`ConfigBuilder::with_single_cert`]

I would like to investigate this sometime later as I DO see some other links like this:

    It is used for functions like [`ClientConfig::builder()`] and [`ServerConfig::builder()`].

Reply:
never mind ... I think we DO want to keep the `()` here, which is both more consistent with reference to `sign()` method later in this paragraph and is more clear since we do not show the path to this method here